Tests/ClientTests/NotificationSurface/OnboardingNotificationCardHelperTests.swift

```diff
@@ -0,0 +1,67 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import XCTest
+
+@testable import Client
+
+class OnboardingNotificationHelperTests: XCTestCase {
+    var nimbusUtility: NimbusOnboardingConfigUtility!
+    typealias cards = NimbusOnboardingConfigUtility.CardOrder
+
+    override func setUp() {
+        super.setUp()
+        DependencyHelperMock().bootstrapDependencies()
+        nimbusUtility = NimbusOnboardingConfigUtility()
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        nimbusUtility.clearNimbus()
```

Comment:
We often `AppContainer.shared.reset()` when we `DependencyHelperMock().bootstrapDependencies()`. I think this could potentially help to make sure the singleton don't stay alive after the test is tear down? Doesn't hurt to put it there.
```suggestion
        super.tearDown()
        AppContainer.shared.reset()
        nimbusUtility.clearNimbus()
```